src/constraints/electric_utility_constraints.jl

```diff
@@ -264,7 +264,9 @@ function add_simultaneous_export_import_constraint(m, p; _n="")
         @constraint(m, NoGridPurchasesBinary[ts in p.time_steps],
             m[Symbol("binNoGridPurchases"*_n)][ts] => {
                 sum(m[Symbol("dvGridPurchase"*_n)][ts, tier] for tier in 1:p.s.electric_tariff.n_energy_tiers) +
-                sum(m[Symbol("dvGridToStorage"*_n)][b, ts] for b in p.s.storage.types.elec) <= 0
+                sum(m[Symbol("dvGridToStorage"*_n)][b, ts] for b in p.s.storage.types.elec) +
```

Comment:
seems unnecessary for dvGridToStorage, dvGridToElectrolyzer, and dvGridToCompressor to be added here since they already have to be <= dvGridPurchase